src/quadratic.jl

```diff
@@ -0,0 +1,66 @@
+"""
+Helper function for quadratic_interval
+"""
+function quadratic_helper(a::Number, b::Number, c::Number, L::AbstractArray)
+
+    Δ = b*b - 4*a*c
+
+    Δ < 0 && return
+
+    Δ = sqrt(Δ)
+
+    if (b >= 0)
+        x0 = -0.5 * (b + Δ)
+
+    else
+        x0 = -0.5 * (b - Δ)
+    end
+
+    if (x0 == 0)
+        x1 = x0
+
+    else
+        x1 = c / x0
+        x0 = x0 / a
+        push!(L, x1)
```

Comment:
And `push!(L, x0, x1)` here.
That way it's clearer that there is a single root in the previous case and 2 here.